src/frontend/components/Chat/SharedChatComponents/ChatMessageContent/index.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { renderImageSnapshot } from 'utils/tests/imageSnapshot';
+import { screen } from '@testing-library/react';
+
+import { ChatMessageContent } from './index';
+
+describe('<ChatMessageContent />', () => {
+  it('renders the message content container and compares it with snapshot', () => {
+    renderImageSnapshot(
+      <ChatMessageContent messageContent={'This is an exemple message'} />,
```

Comment:
```suggestion
      <ChatMessageContent messageContent={'This is an example message'} />,
```

Reply:
correct in commit [f6f606](https://github.com/openfun/marsha/pull/1247/commits/f6f606a391fbf5951e5cb59b9227949d9467522e)